api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -275,4 +280,19 @@ public void removeUserFromGroup(String email, String groupName) {
       return null;
     });
   }
+
+  @Override
+  public boolean isUserMemberOfGroup(String groupName) {
+    return retryHandler.run((context) -> {
+      // There is no endpoint in FireCloud for checking whether a user is a member of a particular
+      // group; so instead, fetch all the group memberships. (There won't be that many for our
+      // users anyway.)
+      for (ManagedGroupAccessResponse group : endUserGroupsApiProvider.get().getGroups()) {
+        if (group.getGroupName().equals(groupName) && group.getRole().equals("member")) {
```

Comment:
What are the other roles? Presumably there are more privileged roles - should we be accepting those as well?

Reply:
I don't know what the other roles are, but we don't use them and probably shouldn't accept them... the expectation is the user had addUserToGroup() called before this gets invoked. 

Changed to use a constant here for "member".